modules/topicsFpdModule.js

```diff
@@ -251,6 +258,25 @@ function loadTopicsForBidders() {
         setLoadedIframeURL(new URL(iframeURL).origin);
         iframeURL && window.document.documentElement.appendChild(ifrm);
       }
+
+      if (bidder && fetchUrl) {
+        let storedSegments = new Map(safeJSONParse(coreStorage.getDataFromLocalStorage(topicStorageName)));
+        const bidderLsEntry = storedSegments.get(bidder);
+
+        if (!bidderLsEntry || (bidderLsEntry && isCachedDataExpired(bidderLsEntry[lastUpdated], fetchRate || DEFAULT_FETCH_RATE_IN_DAYS))) {
+          window.fetch(`${fetchUrl}?bidder=${bidder}`, {browsingTopics: true})
+            .then(response => {
+              return response.json();
+            })
+            .then(data => {
+              if (data && data.segment && !isEmpty(data.segment.topics)) {
+                const {domain, topics, bidder} = data.segment;
+                const fetchTopicsData = getTopicsData(domain, topics)[0];
```

Comment:
if this is just the first topic, you are correct that it is a bug

Reply:
Just pushed a fix for the comment above.